src/jg/coop/sync/guide_sponsors.py

```
@@ -31,47 +30,49 @@ def main(channel_id: int, cache_key: str, cache_days: int, force: bool):
+        *(build_sponsor_args(sponsor) for sponsor in sponsors),
+    ]
+    await sync_guide_channel(client, channel_id, message_args_list)
+
+
+def build_header_args() -> dict:
+    return dict(
+        content=(
+            "# Seznam sponzorů\n\n"
+            "Tyto organizace se podílejí na financování junior.guru. "
+            "Můžeš se tady prokliknout na jejich stránky. "
+            "Sponzorství neznamená, že junior.guru doporučuje konkrétní kurzy, nebo že na ně nemáš psát recenze v klubu. "
```

Ruff (0.15.15)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)